posthog/tasks/test/test_process_event.py

```diff
@@ -14,9 +15,8 @@ def test_capture_new_person(self) -> None:
         ActionStep.objects.create(action=action1, selector='a', event='$autocapture')
         action2 = Action.objects.create(team=self.team)
         ActionStep.objects.create(action=action2, selector='a', event='$autocapture')
-        team_id = self.team.pk
 
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(21):
```

Comment:
What are the extra queries. I assume one is checking existence of person, which makes sense

Reply:
Diffing the captured queries:
Very first query is different which is a DECLARE. I'm not sure where it's coming from.
```
1. DECLARE "_django_curs_140351581124416_sync_48" NO SCROLL CURSOR WITHOUT HOLD FOR SELECT "posthog_user"."id", "posthog_user"."password", "posthog_user"."last_login", "posthog_user"."is_superuser", "posthog_user"."first_name", "posthog_user"."last_name", "posthog_user"."is_staff", "posthog_user"."is_active", "posthog_user"."date_joined", "posthog_user"."email", "posthog_user"."temporary_token", "posthog_user"."distinct_id", "posthog_user"."email_opt_in" FROM "posthog_user" INNER JOIN "posthog_team_users" ON ("posthog_user"."id" = "posthog_team_users"."user_id") WHERE "posthog_team_users"."team_id" = 1676
```